src/components/Icon/FinishIcon.tsx

```
@@ -0,0 +1,32 @@
+  return (
+    <svg
+      width={size}
+      height={size}
+      viewBox="0 0 100 100"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={`${disabled ? "cursor-not-allowed" : "cursor-pointer"} ${className || ""}`}
+      aria-label="완료"
+      onClick={disabled ? undefined : onClick}
+    >
+      <path
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)